pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -69,18 +68,24 @@ func (b *k8sBindingInjector) Inject(binding *servicecatalog.Binding, cred *broke
 }
 
 func (b *k8sBindingInjector) Uninject(binding *servicecatalog.Binding) error {
-	return fmt.Errorf("Not implemented")
+	// TODO: don't hardcode this namespace. https://github.com/kubernetes-incubator/service-catalog/issues/162
```

Comment:
dependent response being, if it's a new argument, add it, even if it's unused.

I think we would pull it out of the ObjectMeta in the binding.

No action necessary.